### PR TITLE
Readd "Use getChunkIfLoadedImmediately in places"

### PR DESCRIPTION
**Pull request by PaulBGD**
I assume the "removed entity stuff" was referring to activation range, but it's unneeded as entities aren't tied to chunks anymore.

### CODE REVIEW

**Review by Machine-Maker — APPROVED** [review]

**Review by MiniDigger — APPROVED** [review]

File: patches/server/0724-Use-getChunkIfLoadedImmediately-in-places.patch [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
++
+     public ResourceKey<DimensionType> getTypeKey() {
+         return this.typeKey;
+     }
+@@ -1366,7 +1373,7 @@ public abstract class Level implements LevelAccessor, AutoCloseable {
+ 
+         for (int l1 = j; l1 <= l; ++l1) {
+             for (int i2 = k; i2 <= i1; ++i2) {
+-                LevelChunk chunk = this.getChunkSource().getChunkNow(l1, i2);
++                LevelChunk chunk = (LevelChunk)this.getChunkIfLoadedImmediately(i1, j1); // Paper
```
Comment (kennytv) [inline patches/server/0724-Use-getChunkIfLoadedImmediately-in-places.patch]:
```suggestion
+                LevelChunk chunk = (LevelChunk) this.getChunkIfLoadedImmediately(l1, i2); // Paper
```

**Review by kennytv — DISMISSED** [review]
2 good approving reviews I see <( ‵□′)───C＜─___-)

File: patches/server/0724-Use-getChunkIfLoadedImmediately-in-places.patch (hunk 1 above)
Comment (PaulBGD) [reply patches/server/0724-Use-getChunkIfLoadedImmediately-in-places.patch]: smh leaf

Comment (JRoy) [reply patches/server/0724-Use-getChunkIfLoadedImmediately-in-places.patch]: not leaf, you just didn't port it to the new the mappings correctly :p

Comment (PaulBGD) [reply patches/server/0724-Use-getChunkIfLoadedImmediately-in-places.patch]: It was already ported, I'll get it fixed

Comment (JRoy) [reply patches/server/0724-Use-getChunkIfLoadedImmediately-in-places.patch]: it was auto-remapped, which doesn't always get variable names correct, but I digress.

Comment (PaulBGD) [reply patches/server/0724-Use-getChunkIfLoadedImmediately-in-places.patch]: I missed the var names myself, I only saw the missing formatting whoops.

**Comment by PaulBGD** [comment]
Bump! Still looks good and would help with the main thread chunk loading deadlocks.

**Review by electronicboy — APPROVED** [review]

**Pull request merged**